src/client/src/rt-theme/colors.ts

```diff
@@ -194,7 +195,8 @@ export const accents: AccentPaletteMap = {
   bad: {
     base: red.base,
     1: red.D1,
-    2: red.L5
+    2: red.L5,
+    3: red.L2
```

Comment:
This exists? — we should ask Carlos and Ed to update the style guide to match. 

Designers always need push back to restrain inconsistency and make sure we're thinking systematically about each addition.

Each addition should be thought of as a future tax on designers and developers trying to make the right choice and use a consistent visual language.